sdk/storage/azure-storage-blob-stress/templates/stress-test-job.yaml

```
@@ -12,39 +12,52 @@ spec:
     - name: fault-injector
       image: azsdkengsys.azurecr.io/stress/httpfaultinjector
       imagePullPolicy: Always
+      command: ['sh', '-c']
+      args:
+        - |
+          set -ex;
+          dotnet dev-certs https --export-path /mnt/outputs/dev-cert.pfx;
+          /root/.dotnet/tools/http-fault-injector;
       resources:
         limits:
           memory: 1Gi
           cpu: "1.0"
+      {{- include "stress-test-addons.container-env" . | nindent 6 }}
     - name: main
       image:  {{ .Stress.imageTag }}
       command: ['sh', '-c']
       args:
         - |
           set -exa;
```

```suggestion
          set -xa;
```
If we leave 'e' in then the fault injector won't be killed since we'll early exit before the other lines. We can bring this back in when we can use proper sidecar support (later version of Kubernetes).

---

Looks like there are a few spots like this. Any script that is responsible for cleaning up the fault injector should be changed.